### ISSUE TITLE
how to set service port name

### ISSUE THREAD

**Issue by jiuchongxiao** [rancher/rancher#14814]
**Rancher versions:**
2.0.6

when i new ingress in load balancing for workloads,it will product a ingress and a service discovery.
the service discovery's yaml will as below:
```
  ports:
  - nodePort: 32399
    port: 9102
    protocol: TCP
    targetPort: 9102
```

but i expected it be:
 ```
 ports:
  - nodePort: 32399
    name: http
    port: 9102
    protocol: TCP
    targetPort: 9102
```

i can sovle it by  copy the service yaml file to xxx.yaml and increase the port name,then kubectl apply -f xxx.yaml.
but i want if is it has a method resolve it by rke config file.

**Comment by jiuchongxiao** [comment]
i hope the ingress service can be editted.
![default](https://user-images.githubusercontent.com/13093730/43373906-568f8b90-93df-11e8-8f31-136c5b7682c8.png)

**Comment by loganhz** [comment]
Fixed in https://github.com/rancher/ui/pull/2546
It will be released in 2.2
